core/sqf/monitor/linux/clio.cxx

```diff
@@ -377,6 +379,21 @@ Local_IO_To_Monitor::Local_IO_To_Monitor(int pv_pid)
         tmpptr++;
     }
 
+    // Remove the domain portion of the name if any
+    char str1[MPI_MAX_PROCESSOR_NAME];
```

Comment:
Is MPI_MAX_PROCESSOR_NAME the maximum name length? Or one more than the maximum name length? If the former, then la_node_name, la_short_node_name and str1 should be declared as char[MPI_MAX_PROCESSOR_NAME+1] to leave room for a trailing null. (The use of strcpy here made me think of that.)